src/5e-SRD-Subclasses.json

```diff
@@ -1,1225 +1,1786 @@
-	"spells": [{
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/burning-hands",
-			"name": "Burning Hands"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/command",
-			"name": "Command"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/blindness-deafness",
-			"name": "Blindness/Deafness"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/scorching-ray",
-			"name": "Scorching Ray"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/fireball",
-			"name": "Fireball"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/stinking-cloud",
-			"name": "Stinking Cloud"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/fire-shield",
-			"name": "Fire Shield"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/wall-of-fire",
-			"name": "Wall of Fire"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/flame-strike",
-			"name": "Flame Strike"
-		}
-	}, {
-		"prerequisites": [{
-			"name": "Warlock 1",
-			"type": "level",
-			"url": "/api/classes/warlock/levels/1"
-		}],
-		"spell": {
-			"url": "/api/spells/hallow",
-			"name": "Hallow"
-		}
-	}],
-	"subclass_levels": {
-		"url": "/api/subclasses/fiend/levels"
-	},
-	"url": "/api/subclasses/fiend"
-}, {
-	"index": "evocation",
-	"class": {
-		"url": "/api/classes/wizard",
-		"name": "Wizard"
-	},
-	"name": "Evocation",
-	"subclass_flavor": "Arcane Tradition",
-	"desc": ["You focus your study on magic that creates powerful elemental effects such as bitter cold, searing flame, rolling thunder, crackling lightning, and burning acid. Some evokers find employment in military forces, serving as artillery to blast enemy armies from afar. Others use their spectacular power to protect the weak, while some seek their own gain as bandits, adventurers, or aspiring tyrants."],
-	"subclass_levels": {
-		"url": "/api/subclasses/evocation/levels"
-	},
-	"url": "/api/subclasses/evocation"
-}]
+[
```

Comment:
Can you make a separate PR with the reformatting so it's not muddled with these changes?

Reply:
It's literally just whitespace changes, does it really warrant its own PR?

Reply:
Let me take another look. It could be fine. Though usually, I just do a separate PR when I do automatic formatting changes since they're basically no-op.

Reply:
@fergcb I can mostly make it out but it isn't pleasant to review. It's not just whitespace changes. There's a bunch of new lines as well, which the normal `hide white spaces` setting doesn't  get rid of.

Reply:
I'll just merge in the format myself so you don't have to bother with another PR.

Reply:
Done! Just merge in the changes from master.

Reply:
....unless that just added a bunch of conflicts for you.

Reply:
I was including line breaks as whitespace. My point being that no structures have changed, only text formatting. There's not really anything to review in that specific diff (unless you were looking at the entire PR diff, rather than individual commit?). It would be just as challenging/simple to review as a separate PR.

Nevermind. If you've already made the change to master I'll figure out how to replay my commits over it in the morning...

Reply:
That looks fine! I'll undo what I did so no conflicts.